.github/workflows/containers.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Containers
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-fedora-infra:
+    name: "Build container image (fedora-infra)"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Build container image (fedora-infra)
+        uses: docker/build-push-action@v2
+        with:
+          push: false
```

Comment:
Doesn't `push` [default to `false`](https://github.com/marketplace/actions/build-and-push-docker-images#customizing) already?

Reply:
Good catch! I dropped it then.